blocks/library/button/index.js

```diff
@@ -45,7 +45,10 @@ registerBlockType( 'core/button', {
 			type: 'string',
 			default: 'none',
 		},
-		color: {
+		textColor: {
+			type: 'string',
+		},
+		backgroundColor: {
```

Comment:
Changing the attribute's name make the "old" buttons block for existing posts invalid, Maybe we should just keep the name as is